src/Keyboard.js

```diff
@@ -35,14 +34,19 @@ export default class Keyboard extends PureComponent {
 		this.handleSymbolsClick = this.handleSymbolsClick.bind(this);
 
 		this.state = {
-			currentLanguage: props.defaultLanguage,
+			currentLanguage: 0,
+			languages: props.languages ==null ? [LatinLayout,CyrillicLayout] : this.props.languages,
```

Comment:
`languages` value never changes inside component, so there is no sense to place it in the state. 
I would place this value in `props`, and `[LatinLayout,CyrillicLayout]` is its defaultProp